.github/renovate.json

```diff
@@ -1,13 +1,9 @@
 {
-  "enabledManagers": ["helm-requirements","gradle-wrapper","regex"],
-  "extends": [":dependencyDashboard"],
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": ["config:base"],
   "labels": ["dependencies"],
-  "helm-requirements":
-  {
-    "fileMatch": ["\\Chart.yaml$"],
-    "aliases": {
-      "hmctspublic": "https://hmctspublic.azurecr.io/helm/v1/repo/"
-    }
+  "helmv3": {
+    "bumpVersion": "patch"
```

Comment:
Seems to work although the version is a bit weird

![image](https://user-images.githubusercontent.com/21194782/214290170-e85df17e-0c88-45fd-9844-4b2179c2942d.png)